resources/org/centos/contra/Infra/topologyTemplates/beaker.template

```diff
@@ -5,7 +5,7 @@ ${index}_${providerType}:
       - resource_group_name: "beaker-slaves"
         resource_group_type: "beaker"
         resource_definitions:
-          - role: "bkr_server"<% if ( jobGroup ) "\n        job_group: \"${jobGroup}\"" %>
+          - role: "bkr_server"<% if ( jobGroup ) out.print "\n            job_group: \"${jobGroup}\"" %>
```

Comment:
is this `out.print` required?

Reply:
Yes - you can see it at play when you run the tests. Without it, the job_group is never injected into the topology, which is why I had to update the test to include the job_group. See for details: https://github.com/openshift/contra-hdsl/pull/86/files#diff-f76c137b127faa4000a9cadc59ead19dR225

Reply:
ack